stardis/opacities/broadening.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+
+
+def calc_phi(delta_nu, line_id, T, splasma):
```

Comment:
As per naming conventions in PEP8, we can't start variable names with capital letter. Perhaps, rename it to `temperature`?

Reply:
I will fix this as a part of my fix of issue #13.